### DISCUSSION TITLE
最新版本闪退啊

### DISCUSSION THREAD

**Discussion by apple314507871** [getlantern/lantern-forum-cn#549] [General]
昨天重装的系统，今天安装最新的版本，一直闪退啊。

**Comment by apple314507871** [comment]
到底能不能解决哦 ，4，5个版本都出现了闪退。要不就是不能翻墙。开了会员4个月，没几天能正常用的。

**Comment by MaxMKMax** [comment]
@apple314507871 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

请问您使用那个电脑手机型号? 和那个系统?

**Reply by apple314507871** [reply]
闪退了，怎么获取日志？  设置网页都进不去

**Reply by apple314507871** [reply]
闪退了，怎么获取日志？  设置网页都进不去

**Reply by apple314507871** [reply]
我用的windows  10    华硕的笔记本

**Comment by MaxMKMax** [comment]
@apple314507871 最新版本应该没有闪退的. 请您是看重新下载. 谢谢!

Flutter 桌面应用程序测试版（8.0.0 版本）
Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb

---
整体下载链接:
https://github.com/getlantern/lantern

https://lantern.io/zh-Hans/faq
如果还遇到什么问题, 您可以参考上面的链接. 谢谢您.

**Reply by apple314507871** [reply]
不闪退了，但是无法通过邮箱和pin码升级到专业版。   PC端

**Reply by MaxMKMax** [reply]
@apple314507871 
您好. 明白的. 那就好!
请您提供我账号邮箱和支付单据. 我帮您提交问题. 谢谢.

**Reply by apple314507871** [reply]
![微信图片_20240614160431](https://github.com/getlantern/lantern-forum-cn/assets/171996799/69639fc4-e468-455a-b43e-193b2f1e3c2a)

**Comment by apple314507871** [comment]
邮箱：314507871@qq.com
